### PR TITLE
Add new version of database

### PR DESCRIPTION
**Pull request by lilaspourpre**

### CODE REVIEW

File: code/backend/database/write_to_bd.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+
+from code.web.db_classes.db import DBaseRusNLP
+from code.web.db_classes.db_writer import WriterDBase
+
+if __name__ == '__main__':
+    if len(sys.argv) < 4:
+        raise Exception("Please specify paths: <data-txt> <db name> <metadata path>")
+    basepath = sys.argv[1]  # "D://syncthing//RusNLP//data-txt"
```
Comment (akutuzov) [inline code/backend/database/write_to_bd.py]: Strange path in the comment

File: code/tests/test_db_reader.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
+import json
+import os
+
+from code.web.db_classes.db import DBaseRusNLP
+from code.web.db_classes.db_writer import WriterDBase
+from code.web.db_classes.db_reader import ReaderDBase
+from helper import *
+
+db = DBaseRusNLP("..///backend///database///rusnlp2.0.db", "..///backend///database///metadata.json")
```
Comment (akutuzov) [inline code/tests/test_db_reader.py]: Why 3 slashes?

File: code/tests/test_db_reader.py [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+import os
+
+from code.web.db_classes.db import DBaseRusNLP
+from code.web.db_classes.db_writer import WriterDBase
+from code.web.db_classes.db_reader import ReaderDBase
+from helper import *
+
+db = DBaseRusNLP("..///backend///database///rusnlp2.0.db", "..///backend///database///metadata.json")
+reader = ReaderDBase(db)
+base_path = "<base path>"  # "D:/syncthing/RusNLP/data-txt/"
```
Comment (akutuzov) [inline code/tests/test_db_reader.py]: Strange path in the comment again

File: code/tests/test_db_reader.py [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+title2hash = create_title2hash(os.path.join(base_path, 'hash_title_url.tsv'))
+hash2url = create_hash2url(os.path.join(base_path, 'hash_title_url.tsv'))
+
+
+def test_title_by_id():
+    for (title, conference, year), hash in title2hash.items():
+        new_titles = reader.select_title_by_id(hash)
+        if reader.select_language_by_id(hash) == 'en':
+            assert len(new_titles) == 1, f"{new_titles}, {title}, {hash}"
```
Comment (akutuzov) [inline code/tests/test_db_reader.py]: Please, no f-strings.
We are using Python 3.5

**Review by akutuzov — CHANGES_REQUESTED** [review]
Please address the comments

**Comment by lilaspourpre** [comment]
relative/absolute paths in sqlite require 3 ///'s in the beginning of the path

https://gist.github.com/ekiara/7676136
https://www.reddit.com/r/learnpython/comments/2jp4jy/sqlite_path_problems_with_sandman_absolute_vs/

**Review by akutuzov — APPROVED** [review]

**Pull request merged**